mgmtsystem_nonconformity/views/mgmtsystem_action.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
```suggestion
```

There is no need to add those lines. Thanks!

Reply:
The documentation says: 
https://www.odoo.com/documentation/16.0/es/developer/reference/backend/data.html
for all XML files.

Reply:
On contrary, Maxime, they must be there. XML format specifies that. If they are not there, they are working by chance, and there's the possibility of a future version of `lxml` (or the low-level library used by Odoo in the future) doesn't parse it correctly. There can be also problems with certain characters if the encoding is not specified.

Reply:
Ok